src/Prototype/ACL/Types.hs

```diff
@@ -25,7 +25,14 @@ import           Servant.API                    ( FromHttpApiData
 
  -- | ID of a group.
 newtype GroupId = GroupId { _unGroupId :: NE.NonEmptyText }
-                deriving (Eq, Show, Ord, ToJSON, FromJSON, ToHttpApiData, FromHttpApiData) via NE.NonEmptyText
+                deriving ( Eq
```

Comment:
I guess I would `fixup` this commit into the relevant commit (and make the "merge remote into local" disappear).